Makefile

```diff
@@ -18,7 +18,7 @@ else ifeq ($(OS),Darwin)
 	NPROCS := $(shell sysctl -n hw.ncpu)
 endif # $(OS)
 
-kcoin:
+kcoin: generate-contract-bindings
```

Comment:
make bindings

Reply:
Do you want me to change the name from generate-contract-bindings -> bindings?